src/containers/Collection.js

```diff
@@ -28,6 +24,7 @@ Collection.propTypes = {
 const mapStateToProps = state => ({
     editor: state.editor,
     user: state.user.user,
+    userSettings:state.user.settings,
```

Comment:
add a space between the colon and `state`.  

Reminding myself to update esdoc rules on this @kdvalin.